### DISCUSSION TITLE
q-btn how to toggle (boolean attributes) state disabled/loading dynamically ?

### DISCUSSION THREAD

**Discussion by robert-hoffmann** [quasarframework/quasar#13009] [General - Components / Directives / etc] [answered]
Sorry if this is a simple question (new to  vue/quasar)

This does not seem to work

    <q-btn"
    round
    color="primary"
    label="OK"
    :disabled="isValid"
    ></q-btn>

    function isValid() {
        return ...some logic (true/false)
    }

Seems like the only way to make this work is to have the disabled attribute present or not, but i have no clue how to toggle it programmatically

Thanks for any help

**Comment by MilosPaunovic** [comment, marked as answer]
Prop name is `disable`, without the letter `D` at the end.

**Reply by robert-hoffmann** [reply]
you are right 

apparently both work when :binding

**Comment by ontwikkelfabriek** [comment]
@robert-hoffmann  @MilosPaunovic 

No idea if it still is valid for Vue 3 but in Vue 2 you had to: 

```
:disable="isValid()"
```

instead of :
```
:disable="isValid"
```

**Reply by robert-hoffmann** [reply]
got it working by binding it to a ref()
But your solution works too

Thanks !
